code/modules/reagents/Chemistry-Machinery.dm

```diff
@@ -1445,3 +1446,47 @@ obj/machinery/computer/pandemic/proc/replicator_cooldown(var/waittime)
 		ui = new(user, src, ui_key, "chem_heater.tmpl", "ChemHeater", 350, 270)
 		ui.set_initial_data(data)
 		ui.open()
+
+///////////////////////////////////////////////////////////////////////////
+
+/obj/machinery/chem_dispenser/soda

```

Comment:
Nitpicking but please rename it into something like /drinks at least instead of /soda
Or theres a possibility to end up with thing similar to obj/structure/stool/bed/chair, which is unpleasant to look at